src/PKSim.Core/Repositories/IIndividualParametersSameFormulaOrValueForAllSpeciesRepository.cs

```diff
@@ -7,6 +8,7 @@ public interface IIndividualParametersSameFormulaOrValueForAllSpeciesRepository
    {
       bool IsSameFormula(ParameterMetaData parameterMetaData);
       bool IsSameValue(ParameterMetaData parameterMetaData);
-      (bool IsSameFormula, bool IsSameValue) IsSameFormulaOrValue(ParameterMetaData parameterMetaData);
+      (bool isSame, bool exists) IsSameFormulaOrValue(ParameterMetaData parameterMetaData);
```

Comment:
while I don't care about Formula or value, I need to know if this exists or not. false, false is not enough